services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentServiceITest.kt

```diff
@@ -221,11 +224,13 @@ class ApplicationAttachmentServiceITest(
             assertThat(attachmentInDb.createdAt).isRecent()
             assertThat(attachmentInDb.applicationId).isEqualTo(application.id)
             assertThat(attachmentInDb.attachmentType).isEqualTo(typeInput)
+            assertThat(attachmentInDb.blobLocation!!.startsWith("${application.id!!}/")).isTrue()
```

Comment:
There's a more specific assertion for this, which gives a better error message if it fails.
```suggestion
            assertThat(attachmentInDb.blobLocation).isNotNull().startsWith("${application.id!!}/")
```
This gives an error like `expected to start with:<"1/"> but was:<"2/e71729a3-3f11-4744-848c-548b5fb99c3d">` instead of the generic `expected to be true`.